### PR TITLE
fixes from QA1, 

### PR DESCRIPTION
**Pull request by TuncAlt**
[US1](https://github.com/TuncAlt/capstone-project/issues/2)
error messages are displayed under input field, 
User gets Submit message, 
min and max temp  range implemented, 
select device option is not an option anymore

### CODE REVIEW

**Review by pung-one — APPROVED** [review]
looks fine!

File: package.json [hunk 1]
```diff
@@ -10,17 +10,18 @@
...
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-hook-form": "^7.43.5",
     "react-icons": "^4.8.0",
-    "react-native-linear-gradient": "^2.6.2",
-    "react-use": "^17.4.0",
     "styled-components": "^5.3.8",
-    "uuid": "^9.0.0"
+    "uid": "^2.0.1",
+    "use-local-storage": "^2.3.6",
```
Comment (Kheirah) [inline package.json:23]: The package `use-local-storage` isn't used within your project and I wonder if you even need it. `uid` hasn't been used either.

**Review by Kheirah — APPROVED** [review]
Looks good overall!

File: package.json (hunk 1 above)
Comment (TuncAlt) [reply package.json:23]: yes you're right, I installed and used the packages before I used the form-hook, I installed use-local-storage accidentally, i fix that

**Review by SarahWett — APPROVED** [review]

**Pull request merged**